box/box.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Start multiple processes and output their logs to the console
+gateway() {
+  echo "Starting Gateway..."
+  ./box/gateway.sh | tee gateway.log
+}
+
+orchestrator() {
+  echo "Starting Orchestrator..."
+  ./box/orchestrator.sh | tee orchestrator.log
+}
+
+mediamtx() {
+  echo "Starting MediaMTX..."
+  ./box/mediamtx.sh | tee mediamtx.log
+}
+
+# Run processes in the background
+gateway &
+orchestrator &
+mediamtx &
+
+# Wait for all background processes to finish
+wait
```

Comment:
Actually not sure what `wait` will do if one of the background processes fail 🤔 
Ideally we should exit this as soon as any command exits tho. WDYT?